JuiceMaker/JuiceMaker/Controller/MakeJuiceViewController.swift

```diff
@@ -0,0 +1,97 @@
+    @IBOutlet private weak var orderPineappleJuiceButton: UIButton!
+    @IBOutlet private weak var orderKiwiJuiceButton: UIButton!
+    @IBOutlet private weak var orderStrawberryAndBananaJuiceButton: UIButton!
+    @IBOutlet private weak var orderMangoJuiceButton: UIButton!
+    @IBOutlet private weak var orderMangoAndKiwiJuiceButton: UIButton!
+    
+    @IBOutlet private weak var editFruitsButton: UIBarButtonItem!
+    
+    @IBOutlet private weak var strawberryLabel: UILabel!
+    @IBOutlet private weak var bananaLabel: UILabel!
+    @IBOutlet private weak var pineappleLabel: UILabel!
+    @IBOutlet private weak var kiwiLabel: UILabel!
+    @IBOutlet private weak var mangoLabel: UILabel!
+    
+    private let juiceMaker = JuiceMaker()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        refreshFruits()
+    }
+    
+    @IBAction private func orderButton(_ sender: UIButton) {
+        switch sender {
+        case orderStrawberryJuiceButton:
+            let result = juiceMaker.make(.strawberry)
+            alert(result)
+        case orderBananaJuiceButton:
+            let result = juiceMaker.make(.banana)
+            alert(result)
+        case orderPineappleJuiceButton:
+            let result = juiceMaker.make(.pineapple)
+            alert(result)
+        case orderKiwiJuiceButton:
+            let result = juiceMaker.make(.kiwi)
+            alert(result)
+        case orderStrawberryAndBananaJuiceButton:
+            let result = juiceMaker.make(.strawberryAndBanana)
+            alert(result)
+        case orderMangoJuiceButton:
+            let result = juiceMaker.make(.mango)
+            alert(result)
+        case orderMangoAndKiwiJuiceButton:
+            let result = juiceMaker.make(.mangoAndKiwi)
+            alert(result)
+        default:
+            return
+        }
+    }
+    
+    private func alert(_ result: Result<FruitJuice, FruitError>) {
+        switch result {
+        case .success(let juice):
+            let successAlert = UIAlertController(title: nil,
+                                                 message: "\(juice.name) 쥬스 나왔습니다! 맛있게 드세요!",
+                                                 preferredStyle: UIAlertController.Style.alert)
+            let confirmAction = UIAlertAction(title: "확인", style: .cancel) { action in
+                self.refreshFruits()
+            }
+            successAlert.addAction(confirmAction)
+            self.present(successAlert, animated: true, completion: nil)
+        case .failure(let error):
+            let failureAlert = UIAlertController(title: nil,
+                                                 message: error.errorDescription,
+                                                 preferredStyle: .alert)
+            let cancelAction = UIAlertAction(title: "아니오", style: .destructive, handler: nil)
+            let presentEditFruitsViewControllerAction = UIAlertAction(title: "예", style: .default) { action in
```

Comment:
클로저 또는 for...in과 같은 흐름제어 구문에서 주어진 상수를 사용하지 않을 것이라면 [Wildcard pattern](https://docs.swift.org/swift-book/ReferenceManual/Patterns.html#ID420)을 적용할 수 있습니다. 이를 통해 이 코드를 읽는 사람에게 이 코드 블럭에서 이 상수는 사용되지 않는구나하는 정보를 알려줄 수 있어요.

Reply:
77, 78번 라인에 있는 상수들은 바로 밑에서 
failureAlert.addAction(cancelAction)
failureAlert.addAction(presentEditFruitsViewControllerAction)
이렇게 쓰이고 있습니다. 혹시 제가 이해를 잘못 한걸까요?

Reply:
UIAlertAction에서 등장한 action은 사용되고 있지 않은 것 같네요 😉

```suggestion
            let presentEditFruitsViewControllerAction = UIAlertAction(title: "예", style: .default) { _ in
```

Reply:
ㅎㅎ... 바로 고치겠습니다